recipes/libcamera/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+os_version:
+  - linux_aarch64: alma8
```

Comment:
do you really need this? this seems risky, lets migrate aarch later?

Reply:
I actually added this because I was trying to fix an error with `memfd_create`. Which in the latest build seems indeed fixed (but there are other issues...